index.html

```diff
@@ -248,45 +248,45 @@ <h1>ライフスタイルに合わせて選べる多彩なプラン。</h1>
               ビジター料金でのご利用も可能ですので、月額会員登録の前に体験をしたいという方もぜひご利用ください。
             </p>
           </div>
-          <div class="list row">
+          <div class="row">
             <div class="col-12 col-md-4 mb-3 mb-md-0">
-              <table>
-                <tr><th>デイタイムプラン</th></tr>
-                <tr><td>
+              <ul>
+                <li>デイタイムプラン</li>
+                <li>
                   <p><span>￥</span>5,000</p>
                   <p>月額</p>
-                </td></tr>
-                <tr><td>入会金0円</td></tr>
-                <tr><td>平日：9時～17時</td></tr>
-                <tr><td>土日祝：利用ナシ</td></tr>
-                <tr><td>コアタイムでの利用に最適なプラン</td></tr>
-              </table>
+                </li>
+                <li>入会金0円</li>
+                <li>平日：9時～17時</li>
+                <li>土日祝：利用ナシ</li>
+                <li>コアタイムでの利用に最適なプラン</li>
+              </ul>
             </div>
-            <div class="col-12 col-md-4 mb-3 mb-md-0">
-              <table>
-                <tr><th>スタンダードプラン</th></tr>
-                <tr><td>
+            <div class="standard col-12 col-md-4 mb-3 mb-md-0">
+              <ul>

```

Comment:
これなら265行目のdivがいらなそうですね

```suggestion
              <ul class="standard col-12 col-md-4 mb-3 mb-md-0">
```

これに伴って以下の修正も必要そうです！